public class TestException {
    public static void main(String[] args) {

        System.out.println("hello arjun");
        System.out.println("how are u ");
        System.out.println(10/0);//risky code hain - jiski wajahh se exception ata hain

        System.out.println("i am godd");
    }
// isme apnacode to comple ho gaya hoga ,but runtime pe exception

}
